.github/pull_request_template.md

```diff
@@ -51,11 +51,14 @@ and would like to solicit the team's feedback. Optionally, provide background
 information regarding your questions and concerns. -->
 
 ### Acceptance Validation
-
 <!-- Were you able to fully test the acceptance criteria on the related ticket? if not, why not? -->
 
 <!-- Insert screenshots if applicable (drag images here) -->
 
+<!-- Did you deploy this feature branch to the AWS `dev` environment?  https://bcda-ci.adhocteam.us/job/BCDA%20-%20Deploy/build 
```

Comment:
Should the link point to the Jenkins job specific to deploying to `dev`?

Reply:
Yes, this might be the wrong link.  I always use the [Build and Package](https://bcda-ci.adhocteam.us/job/BCDA%20-%20Build%20and%20Package/build) job since its parameters are at the level I'm usually operating: what branch(es) should be deployed.  Personally, I've never used the Jenkins job specifically deploying to `dev` because it requires specifying an AMI.

Reply:
Oh, good point.  You are exactly right, we should def link to `Build and Package` instead of the `Deploy` job... need to build the AMI for the branch first!